Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:alpine3.10
+LABEL author "Wes Lambert, wlambertts@gmail.com"
+LABEL description="Dockerised version of Cyberchef server (https://github.com/gchq/CyberChef-server)"
+LABEL copyright "Crown Copyright 2020"
+LABEL license "Apache-2.0"
+RUN apk update && apk add --no-cache --update npm && \
```

Comment:
`npm` comes as part of the node alpine image, please check whether this extra install is necessary.

Reply:
I tried without adding/updating, and it didn't seem to work for me.